sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/jackson/ClientRegistrationDeserializer.java

```diff
@@ -53,7 +53,7 @@ public ClientRegistration deserialize(JsonParser parser, DeserializationContext
             .authorizationGrantType(
                 AUTHORIZATION_GRANT_TYPE_CONVERTER.convert(
                     findObjectNode(clientRegistrationNode, "authorizationGrantType")))
-            .redirectUriTemplate(findStringValue(clientRegistrationNode, "redirectUriTemplate"))
+            .redirectUri(findStringValue(clientRegistrationNode, "redirectUri"))
```

Comment:
Hi, @alzimmermsft , 
I think you do not need to update it.
Because after spring-boot's version upgrade to 2.4.0, the spring-security will update to 5.4.1, I'll use `ClientRegistrationDeserializer.java` in spring-security, and delete all contents in package `com.azure.spring.aad.webapp.jackson`

cc @yiliuTo , FYI>